pages/start.tsx

```diff
@@ -113,7 +113,7 @@ interface LoginStep2Props {
 
 const LoginStep2: React.FC<LoginStep2Props> = ({
   handleLoginStep,
-  handleUpdateInterests: handleUpdateInterestsParent
+  handleUpdateInterests: handleUpdateInterestsParent,
```

Comment:
i think we might need to turn off `trailingComma` in the prettier config

Reply:
I disabled the error in the eslint config. I think trailing commas are good for diffs. Open to suggestion though